diesel_cli/src/cli.rs

```diff
@@ -57,6 +57,9 @@ pub fn build_cli() -> App<'static, 'static> {
         ).subcommand(SubCommand::with_name("drop")
             .about("Drops the database specified in your DATABASE_URL.")
             .setting(AppSettings::Hidden)
+        ).subcommand(SubCommand::with_name("touch")
```

Comment:
I suppose it could work out better if all tests assumed that their database may or may not exist. At least some of them require it preexisting (I'd wager most), for example dropping the `touch` in the `test_all` scripts for the examples gives

```
+ cd diesel_infer_schema
+ cargo test --features 'lint postgres'
warning: package replacement is not used: https://github.com/rust-lang/crates.io-index#diesel_infer_schema:0.11.0
warning: package replacement is not used: https://github.com/rust-lang/crates.io-index#diesel_codegen:0.11.0
    Finished dev [unoptimized + debuginfo] target(s) in 0.0 secs
     Running /Users/tschottdorf/rust/diesel/target/debug/deps/diesel_infer_schema-a9de714f71691e1f

running 6 tests
test information_schema::tests::load_table_names_loads_from_public_schema_if_none_given ... FAILED
test information_schema::tests::get_table_data_loads_column_information ... FAILED
test information_schema::tests::load_table_names_loads_from_custom_schema ... FAILED
test information_schema::tests::get_primary_keys_only_includes_primary_key ... FAILED
test information_schema::tests::load_table_names_output_is_ordered ... FAILED
test information_schema::tests::skip_views ... FAILED
```